quick_test.sh

```
@@ -1,52 +1,16 @@
 #!/bin/bash
+# Quick test to verify the gesture coordinator fix
 
-# Quick test script to verify the corrected behavior
+cd /home/daniel/work/puzzgameFlutter
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)